src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -398,6 +399,22 @@ static boolean isSnowpipeIngestion(Map<String, String> config) {
             .equalsIgnoreCase(IngestionMethodConfig.SNOWPIPE.toString());
   }
 
+  /**
+   * @param config config with applied default values
+   * @return true when Iceberg mode is enabled.
+   */
+  public static boolean isIcebergIngestion(Map<String, String> config) {
```

Comment:
should be rather `isIcebergIngestionEnabled`? :D

Reply:
I will go with simpler `isIcebergEnabled()`